crates/consensus/primary/src/consensus_bus.rs

```diff
@@ -55,10 +60,16 @@ struct ConsensusBusInner {
     new_certificates: MeteredMpscChannel<Certificate>,
     /// Outputs the sequence of ordered certificates to the primary (for cleanup and feedback).
     committed_certificates: MeteredMpscChannel<(Round, Vec<Certificate>)>,
+
     /// Outputs the highest committed round & corresponding gc_round in the consensus.
-    tx_consensus_round_updates: watch::Sender<ConsensusRound>,
+    tx_committed_round_updates: watch::Sender<Round>,
     /// Hold onto a receiver to keep it "open".
-    _rx_consensus_round_updates: watch::Receiver<ConsensusRound>,
+    _rx_committed_round_updates: watch::Receiver<Round>,
+
+    /// Outputs the highest gc_round from the consensus.
+    tx_gc_round_updates: watch::Sender<AtomicRound>,
```

Comment:
I think putting an atomic in a watch is a bit of an anti-pattern.  What is the purpose?  The watch will give you the latest thing, are you trying to change a value underneath code that has already queried the watch?  It seems like using the latest value when you check the watch should always be sufficient?
Also, being able to take a value from a watch and then changing it via an atomic store is violating the watch contract since subscribers will not be notified so I really think a watch should not contain an atomic.